lib/shared/messages.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
    */
   info: {
 
-    flashComplete: _.template('Your flash is complete!')
+    flashComplete: _.template([
+      '<%= imageBasename %> was written to',
```

Comment:
Could do, but would it look weird if the success window gets too big? Given that it contains the filename of the image and the description of the drive, it might already be quite long?
Mind sharing a screenshot @Shou ? ;)